djstripe/signals.py

```diff
@@ -1,4 +1,13 @@
 # -*- coding: utf-8 -*-
+"""
+.. module:: djstripe.signals.
+
+   :synopsis: dj-stripe - signals are sent for each event Stripe sends to the app
+
+.. moduleauthor:: N/A
```

Comment:
Authors are me and pydanny

https://github.com/kavdev/dj-stripe/blame/%23162-api-updates-through_2015-07-28/djstripe/signals.py